tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -35,6 +35,17 @@ sudo systemsetup -settimezone America/Los_Angeles
 sudo systemsetup -setusingnetworktime on
 date
 
+# See https://github.com/grpc/grpc/issues/14815
```

Comment:
The comments around this code should be clear that this is a _temporary_ workaround, right?

Reply:
Added a TODO